repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -65,26 +65,36 @@ object ProjectUpdateRequest {
 final case class Version(name: String, message: String, target: String, commit: Commit)
 object Version {
   implicit val versionPlayFormat: OFormat[Version] = Json.format[Version]
+  implicit val seqVersionPlayFormat: Format[Seq[Version]] =
+    Format(Reads.seq(versionPlayFormat), Writes.seq(versionPlayFormat))
 }
 
 final case class Commit(id: String)
 object Commit {
   implicit val commitFormat: OFormat[Commit] = Json.format[Commit]
 }
 
-final case class ProjectFile(path: Path, content: String)
-
 object ProjectFile {
-  implicit object ProjectFileFormat extends Format[ProjectFile] {
-    override def reads(json: JsValue): JsResult[ProjectFile] =
-      JsSuccess(ProjectFile(Paths.get((json \ "path").as[String]), (json \ "content").as[String]))
 
-    override def writes(o: ProjectFile): JsValue = JsObject(
-      Seq("path" -> JsString(o.path.toString), "content" -> JsString(o.content))
-    )
+  implicit object pathFormat extends Format[Path] {
+    override def writes(o: Path): JsValue = JsObject(Seq("path" -> JsString(o.toString)))
```

Comment:
```suggestion
    override def writes(o: Path): JsValue = JsString(o.toString)
```
Don't forget to change reads in a similar way

Reply:
Excuse me, are you sure about that? Can you give a little explaination?(some tests failed with that change)

Reply:
What tests have failed? What is the reason of it?
I can write you an explanation, but I suggest you use both of these formatters (before and after changes) and compare the results.

Reply:
`MalformedRequestContentRejection(JsResultException(errors:List((,List(JsonValidationError(List("folder/test.txt" is not an object),WrappedArray())))))`
in **ProjectFileControllerTest:** when **when upload file**
It seems to me that it needs 'path', and jsObject has this structure. But if you sure, I shall try to dig details

Reply:
Look at my first comment.
**_Don't forget to change reads in a similar way_**
Did you do it?

Reply:
I guess I know what you talk about. I will check it

Reply:
got it